### ISSUE TITLE
user feedback "outros e outras"

### ISSUE THREAD

**Issue by danielnaber** [languagetool-org/languagetool#3260]
> "outros e outras" e qualquer sequência igual nos dois gêneros não é redundância

**Comment by jaumeortola** [comment]
Sometimes these rules can be a good style recommendation, but, obviously, it is impossible to know the writer's intention. 
![imatge](https://user-images.githubusercontent.com/4289501/87532478-81e54580-c693-11ea-8125-e112e159487e.png)
In Spanish, these rules have been disabled or removed. They belong to the picky mode.

**Comment by danielnaber** [comment]
> They belong to the picky mode.

Okay, feel free to add `tags="picky"` where appropriate. The tags should inherit from `<category>` to `<rulegroup>` to `<rule>`, i.e. adding a tag to a category should apply it to all rules in it (but please test if that really works).

**Comment by jaumeortola** [comment]
The inheritance from `<rulegroup>` or `<category>` to `<rule>` doesn't work.

**Comment by jaumeortola** [comment]
Added picky tags https://github.com/languagetool-org/languagetool/commit/2634b85e25f848e6d6c6a4e60372bab168cd07bb

**Comment by danielnaber** [comment]
Indeed, I have just committed a fix (https://github.com/languagetool-org/languagetool/commit/f8662cf01867eb8b10fb24b0279997b1c0d5ce08).
